avocado/plugins/replay.py

```diff
@@ -213,11 +213,12 @@ def run(self, args):
                                         for _ in args.multiplex_files]
             else:
                 mux = jobdata.retrieve_mux(resultsdir)
-                if mux is None:
-                    log.error('Source job multiplex data not found. Aborting.')
+                if mux is None:     # Fallback to multiplex_files
+                    log.error("Source job multiplex data not found. "
```

Comment:
yep, this is a remnant from the previous version where I handled it differently. I'll remove this modification in v2.